Extensions/rnaoptions_defaults.hh

```diff
@@ -142,6 +145,11 @@
 	inline static const char* getProbing_normalization() {
 		return "centroid";
 	}
+    inline static std::vector<std::pair<const char*, unsigned> > getInputs() {
+        std::vector<std::pair<const char*, unsigned> > emptyInput;
+        emptyInput.push_back(std::make_pair((char*)"", 0));
+        return(emptyInput);
+    }
```

Comment:
the functions are here redefined for cases where the programmer does *not* use the "perl Misc/Applications/addRNAparameters.pl" program to change the out.mf file. In such a case, e.g. the `->consensusType` is simple nowhere defined and would raise a null pointer exception. 

This is different for the `gapc::Opts::getOpts()->inputs;` function. It is part of the "default" parameter set and thus always defined. Can you try moving both versions out of the IF condition and check if it works?

Reply:
You mean just move the first function
```
inline static std::vector<std::pair<const char*, unsigned> > getInputs() {
        return gapc::Opts::getOpts()->inputs;
}
```
to line 160 (after the first `endif`) and delete the redefinition? 
Or delete both altogether?

I tried the first case and trying to compile the instance `mfepp` afterwards exits with an error saying `gapc::Opts` has not been declared.

I thought the `input` variable is initialized in the Opts class in `rnaoptions.hh` and therefore not accessible whenever the perl script is not called?

Reply:
hm, you might be right. Leave as is, we will take care sometimes else